basic_module/account/src/core.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Kodebox, Inc.
+// This file is part of CodeChain.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+use crate::error::Error;
+use crate::types::SignedTransaction;
+pub use ckey::{Ed25519Private as Private, Ed25519Public as Public, Error as KeyError, Password, Signature};
+pub use coordinator::context::SubStorageAccess;
+pub use coordinator::types::{ErrorCode, TransactionExecutionOutcome};
+
+pub trait CheckTxHandler {
+    fn check_transaction(&self, tx: &SignedTransaction) -> Result<(), ErrorCode>;
+}
+
+pub trait TransactionExecutor {
+    fn execute_transactions(&self, transactions: &[SignedTransaction]) -> Result<Vec<TransactionExecutionOutcome>, ()>;
+}
+
+pub trait AccountManager {
+    fn add_balance(&self, address: &Public, val: u64);
```

Comment:
The name `address` is a mismatch with the value `Public`, and I think the name is inappropriate. In a brief discussion at the office, we decided to change the name `address` to `account_id`. But I forgot about it and didn't make a PR. 

Could you include the changes in this PR after I make the PR?

Reply:
I made a PR #406

Reply:
The PR was merged.

Reply:
okay.

Reply:
@somniumism done